apps/entitree-images/package.json

```diff
@@ -20,45 +20,36 @@
     "@mui/icons-material": "^5.8.2",
     "@mui/material": "^5.8.2",
     "@mui/styles": "^5.8.0",
-    "@next-auth/mongodb-adapter": "^1.0.3",
     "@next-auth/prisma-adapter": "^1.0.3",
     "@prisma/client": "3.14.0",
     "axios": "^0.27.2",
     "canvas": "^2.9.1",
     "form-data": "^4.0.0",
     "lodash": "^4.17.21",
-    "mongodb": "^4.6.0",
-    "mongoose": "^6.3.5",
-    "mongoose-sequence": "^5.3.1",
     "next": "12.1.6",
     "next-auth": "^4.3.4",
     "nextjs-cors": "^2.1.1",
     "node-fetch": "^3.2.5",
     "query-string": "^7.1.1",
-    "ra-core": "^4.1.1",
     "ra-data-simple-prisma": "^0.3.3",
-    "ra-data-simple-rest": "4.1.1",
-    "ra-i18n-polyglot": "^4.1.1",
-    "ra-language-english": "^4.1.1",
+    "ra-i18n-polyglot": "^4.1.0",
+    "ra-language-english": "^4.1.0",
     "react": "18.1.0",
     "react-admin": "4.1.1",
     "react-dom": "18.1.0",
     "react-hook-form": "^7.31.3",
     "react-router": "^6.3.0",
     "react-router-dom": "^6.3.0",
     "schema-dts": "^1.1.0",
-    "sharp": "^0.30.6",
-    "stream": "^0.0.2",
+    "sharp": "^0.30.5",
     "util": "^0.12.4",
     "wikidata-sdk": "^8.0.0"
   },
   "devDependencies": {
-    "@types/jest": "^28.1.0",
-    "@types/react": "18.0.10",
+    "@types/react": "18.0.9",
     "eslint": "8.16.0",
     "eslint-config-next": "12.1.6",
     "jest": "^28.1.0",
-    "prisma": "^3.14.0",
```

Comment:
I think this is still needed, is it not better to have a separate PR for the other apps?